ui/de-lib/src/main/java/org/iplantc/de/desktop/client/presenter/DesktopPresenterImpl.java

```diff
@@ -105,9 +106,23 @@
  */
 public class DesktopPresenterImpl implements DesktopView.Presenter {
 
-
-
-    interface AuthErrors {
+    private final class NewSysMessageCountCallback implements AsyncCallback<Counts> {
+		@Override
+		public void onFailure(Throwable caught) {
+			// TODO Auto-generated method stub
```

Comment:
Not sure if this is still WIP, but we should do something if a failure occurs here I would think.